rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -969,9 +969,8 @@ void fmt_class_string<CellError>::format(std::string& out, u64 arg)
 		STR_CASE(CELL_ESDKVER);
 		STR_CASE(CELL_ENOLICDISC);
 		STR_CASE(CELL_ENOLICENT);
+		default: return unknown;
```

Comment:
That was the point: see the warning and add unimplemented case.

Reply:
Ok, reverted in hopes that it gets implement soon :+1: